.github/workflows/publishimage.yml

```diff
@@ -0,0 +1,70 @@
+name: Publish pre Image from master
+
+# The workflow will push PRE images from master on every merge. The images will be tagged with PRE and the next minor increase on the
+# semver(based on the github releases)
+# It will NOT produce releases and release images.
+
+on:
+  push:
+    branches:
+      - master
+
+# will produce tags in format fiware/orion:<SEMVER>-PRE-<INCREASING-NUMBER>, f.e. fiware/orion:2.6.0-PRE-12
+env:
+  IMAGE_NAME: fiware/orion
+  IMAGE_TAG_PRE: -PRE-${{ github.run_number }}
+
+jobs:
+  deploy-release:
+
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'push'
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
```

Comment:
Buildx? Is this a typo?

Reply:
No, see: https://docs.docker.com/buildx/working-with-buildx/

Reply:
As far as I understand from the link you cite, `buildx` seems to be an experimental feature. Could regular `build` be used instead? Which special feature existing in `buildx` but not in `build` are we needed here?

Reply:
I wasn't aware that its still marked as experimental, I usually use it to be prepared for switching to OCI compliant images. I can switch to the normal docker command, if you prefer that.

Reply:
> I can switch to the normal docker command, if you prefer that.

I think it would be preferable, please.

Reply:
Fixed in e211d6e